tox.ini

```diff
@@ -92,7 +92,8 @@ commands =
       --cov-report html:{envdir}/cov/html {posargs} \
       ./lib/pbench/test/unit/common \
       ./lib/pbench/test/unit/server \
-      ./lib/pbench/test/functional
+      ./lib/pbench/test/functional \
+      ./server/bin/test
```

Comment:
I also like this one

Reply:
If we click-ified them they'd go under the existing `lib/pbench/test` tree, which wouldn't be a bad thing; but there's not much value otherwise in click-ifying server pipeline scripts, and it doesn't seem worth hacking up this one again.

So, yeah, sure: `server/bin/test/unit` is a sensible path.

And I'd love to have someone move `./lib/pbench/test/functional` under `py3-agent` since there are no "pytest functional" server tests and having agent tests run at the end of `py3-server` is just annoying. :-)

Reply:
sure

Reply:
I will make that change